sentinelone_connector.py

```diff
@@ -957,7 +979,9 @@ def _handle_export_threats(self, param):
                     return action_result.set_status(phantom.APP_ERROR, "Did not get proper response from the server")
                 self.save_progress('Agent query: {}'.format(ret_val))
 
-                if ret_val == '0':
+                if ret_val == '-1':
```

Comment:
Hi @abhijeeta-metron,
While testing for the negative scenarios, we observed that the 'export threats' action was passing with invalid parameter values. To handle this case, we have added a check for the ‘-1’ string value after the get_agent_id method call. Similarly, we have added the check for '-1' string in other actions as well. 

Action result status before adding -1 check:
![image](https://user-images.githubusercontent.com/72184309/150536834-3660d45d-be87-494a-a54b-494dad0c165c.png)

Action result status after adding -1 check:
![image](https://user-images.githubusercontent.com/72184309/150536898-3e20f614-545e-46c8-9b1a-6272552afed8.png)

Please let us know if you have any queries regarding this change.